draft-ietf-oauth-selective-disclosure-jwt.md

```diff
@@ -102,11 +102,11 @@ has to verify that all disclosed claim values were part of the original
 Issuer-signed JWT. The Verifier will not, however, learn any claim
 values not disclosed in the Disclosures.
 
-This document also specifies an optional mechanism for Key Binding,
-which is the concept of binding an SD-JWT to a Holder's public key
-and requiring that the Holder prove possession of the corresponding
-private key when presenting the SD-JWT.
-The strength of the binding is conditional upon the trust
+For SD-JWTs where the issuer is binding an SD-JWT to a public key associated to the
+Holder (e.g., using the `cnf` claim [@?RFC7800]), this document also defines a
+format for SD-JWTs with Key Binding (SD-JWT-KB).  By optionally sending an SD-JWT-KB to a
+Verifier, the Holder can prove to the verifier that they hold the private key
+associated to the SD-JWT. The strength of the binding is conditional upon the trust
```

Comment:
It would be better to focus on the fact that the SD-JWT-KB can be used for proving something instead of focusing on the binding in the first sentence.
```suggestion
This document also defines a format for SD-JWTs with Key Binding (SD-JWT-KB).
By optionally sending an SD-JWT-KB to a
Verifier, the Holder can prove to the verifier that they hold the private key
associated to the SD-JWT (e.g., using the `cnf` claim [@?RFC7800]). The strength of the binding is conditional upon the trust
```